### DISCUSSION TITLE
CPDLC AOC integration

### DISCUSSION THREAD

**Discussion by WillyOnGithub** [magknight/787aviatorsTracker#540] [Completed things] [answered]
Add in support for CPDLC clearances enroute

**Comment by aeroniemi** [comment]
At present it doesn't really seem to be that useful - there are very few instances where enroute CPDLC is supported, but we will reconsider it if it appears that usage will become more commonplace

**Comment by aeroniemi** [comment, marked as answer]
This has been done now - marked as closed
